content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,203 @@
+be more expressive than annotations were on Ingress.
+
+
+## Upgrading to Gateway
+
+Migrating from Ingress to the Gateway API may seem intimidating, but luckily Kubernetes just
+released a tool to simplify the process.
+[ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway) assists in the migration by
+converting your existing Ingress resources into Gateway API resources. Here is how you can get
+started with Gateway API and using ingress2gateway:
+
+1. [Install a Gateway
+   controller](https://gateway-api.sigs.k8s.io/guides/#installing-a-gateway-controller) OR [install
+   the Gateway API CRDs manually](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api) .
+
+2. Install [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway).
+   
+   If you have a Go development environment locally, you can install `ingress2gateway` with:
+
+   ```
+   go install github.com/kubernetes-sigs/ingress2gateway@v0.1.0
+   ```
+
+   This installs `ingress2gateway` to `$(go env GOPATH)/bin/ingress2gateway`.
+
+   Alternatively, follow the installation guide
+   [here](https://github.com/kubernetes-sigs/ingress2gateway#installation).
+
+3. Once the tool is installed, you can use it to convert the ingress resources in your cluster to
+   Gateway API resources.
+
+   ```
+   ingress2gateway print
+   ```
+
+   This above command will:
+
+   1. Load your current Kubernetes client config including the active context, namespace and
+      authentication details.
+   2. Search for ingresses and provider-specific resources in that namespace.
+   3. Convert them to Gateway API resources (Currently only Gateways and HTTPRoutes). For other
+   options you can  run the tool with `-h`, or refer to
+   [https://github.com/kubernetes-sigs/ingress2gateway#options](https://github.com/kubernetes-sigs/ingress2gateway#options).
+
+4. Review the converted Gateway API resources, validate them, and then apply them to your cluster.
+
+5. Send test requests to your Gateway to check that it is working. You could get your gateway
+   address using `kubectl get gateway <gateway-name> -n <namespace> -o
+   jsonpath='{.status.addresses}{"\n"}'`.
+
+6. Update your DNS to point to the new Gateway. 
+
+7. Once you've confirmed that no more traffic is going through your Ingress configuration, you can
+   safely delete it.
+
+## Wrapping up
+
+Achieving reliable, scalable and extensible networking has always been a challenging objective. The
+Gateway API is designed to improve the current Kubernetes networking standards like ingress and
+reduce the need for implementation specific annotations and CRDs.
+
+It is a Kubernetes standard API, consistent across different platforms and implementations and most
+importantly it is future proof. The Gateway API is considered the successor to Ingress; Gateway API
+and Ingress2gateway are supported by a dedicated team under SIG Network that actively work on it and
```

Comment:
```suggestion
and `ingress2gateway` are supported by a dedicated team under SIG Network that actively work on it and
```

Reply:
We need to be consistent, its either I do ingress2gateway with lower "i" across all the blog, including the start of sentences. Or i do it with capital "I".
The repo name is with lower i but of course we can change that as well if we decide. 

Re code block, I do not want to put it in code block here. In this context it is "Gateway API and ingress2gateway are supported..." same as I don't put Gateway API in a code block I don't think I need one for ingress2gateway

Reply:
I changed the last two places to be with lower "i", do let me know if you prefer otherwise.